### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use `ldd`.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/capuanob/nweb-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image name.

File: nweb23.c [hunk 3]
```diff
@@ -163,9 +163,11 @@ int main(int argc, char **argv)
     (void)printf("ERROR: Can't Change to directory %s\n",argv[2]);
```
Comment (xansec) [inline nweb23.c:163]: I'm ok with this change.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**